src/main/java/com/mineinabyss/looty/ecs/systems/CooldownDisplaySystem.kt

```diff
@@ -14,7 +14,7 @@ import kotlin.time.ExperimentalTime
 
 private const val INTERVAL = 3L
 
-object CooldownDisplaySystem : TickingSystem(interval = INTERVAL) {
+object CooldownDisplaySystem : TickingSystem(1) {
```

Comment:
Please keep using the INTERVAL constant in here and change it to 1L there instead. It's just generally good practice.